library/src/helpers/createNestedClassName.ts

```diff
@@ -0,0 +1,9 @@
+import { bemClasses } from './bemClasses';
+
+export const createNestedClassName = (
```

Comment:
I don't like this name, as it suggest that you always get classname that includes `nested` word in it